sites/bizbash/components/top-list-galleries.marko

```diff
@@ -0,0 +1,12 @@
+import { buildImgixUrl } from '@base-cms/image';
+import { getAsArray } from '@base-cms/object-path';
+
+$ const connection = getAsArray(input, 'galleries.edges');
+
+<if(connection.length)>
+  $ const galleries = connection.map(({ node }) => node);
+  <for|gallery| of=galleries>
+    <h4 class="top-list__title">${gallery.shortName}</h4>
```

Comment:
The block name should be passed in and then `__title` should be concatenated with it